app/task_routes.py

```diff
@@ -0,0 +1,120 @@
+from flask import Blueprint, jsonify, abort, make_response, request
+from app.models.task import Task
+from app import db
+from datetime import datetime
+import requests
+import os
+
+tasks_bp = Blueprint('tasks', __name__, url_prefix='/tasks')
+
+def send_slack_notification(task):
+    slack_path = 'https://slack.com/api/chat.postMessage'
+    headers = {'Authorization': f'Bearer {os.environ.get("SLACK_API_KEY")}'}
+    
+    query_param = {
+        'channel': 'task-notifications',
+        'text': f'Someone just completed {task}!'
+    }
+
+    return requests.post(slack_path, params=query_param, headers=headers)  
+
+def validate_task(task_id):
```

Comment:
Style: Since this returns a task and not just a boolean I'd probably name it something like `get_task_or_abort`:

```suggestion
def get_task_or_abort(task_id):
```